### PR TITLE
Remove outdated README links and minimize some SVGs

### PR DESCRIPTION
**Pull request by Artem13327**
1. Simplified SVG
2. Slightly corrected the readme

### CODE REVIEW

**Comment by lafriks** [comment]
Very unrelated changes without any description on why they are needed and what they fix/improve

**Comment by delvh** [comment]
I think the title should be `Removed outdated README links and minimized some SVGs`.
Then, everyone knows what you are doing here.

File: docs/README.md [hunk 1]
```diff
@@ -1,8 +1,5 @@
 # Gitea: Docs
 
-[![Join the chat at https://img.shields.io/discord/322538954119184384.svg](https://img.shields.io/discord/322538954119184384.svg)](https://discord.gg/Gitea)
```
Comment (lunny) [inline docs/README.md:3]: Why remove this line?

Comment (Artem13327) [reply docs/README.md:3]: @lunny 
The link to the chat is at https://github.com/go-gitea/gitea/blob/main/README.md. Why duplicate it here

**Comment by silverwind** [comment]
The SVGs are generated with below node scripts. Please apply your configuration changes to SVGO there and then run `make svg generate-images` to regenerate the output files.

https://github.com/go-gitea/gitea/blob/main/build/generate-svg.js
https://github.com/go-gitea/gitea/blob/main/build/generate-images.js

**Review by delvh — APPROVED** [review]

**Comment by lunny** [comment]
> Some of the SVGs are generated via below node scripts. Please apply your configuration changes to SVGO there and then run `make svg generate-images` to regenerate the output files.
> 
> https://github.com/go-gitea/gitea/blob/main/build/generate-svg.js https://github.com/go-gitea/gitea/blob/main/build/generate-images.js

Wait for the changes.

**Pull request closed without merging**